style.css

```diff
@@ -2,47 +2,89 @@
 @import url('https://fonts.googleapis.com/css?family=Muli&display=swap');
 @import url('https://fonts.googleapis.com/css?family=Quicksand&display=swap');
 
-body {
+.body {
   display: flex;
   min-height: 100vh;
 }
```

Comment:
- [ ] Your entire project has some overflow issues that need to be rectified. From the video below you can see that everything isn't well aligned so please do well to fix this as well :wink: 
https://www.loom.com/share/56d26a72f6554bc4afc037c1a240d741